src/components/composite/reactQueryBuilder/PropertyValueEditor.tsx

```
@@ -101,28 +106,33 @@ export function PropertyValueEditor(props: ExpertFilterPropertyProps) {
+                                    placeholder={
+                                        propertyValues?.length > 0 ? '' : intl.formatMessage({ id: 'valuesList' })
+                                    }
+                                />
+                            )}
+                            freeSolo
+                            autoSelect
+                            onChange={(event, value) => {
+                                onChange(FieldConstants.PROPERTY_VALUES, value);
+                            }}
+                            size="small"
+                            filterOptions={useCustomFilterOptions()}
```

ESLint

[error] React Hook "useCustomFilterOptions" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)